styles/plain-language-guidelines/use-simple-words-and-phrases.yml

```diff
@@ -0,0 +1,246 @@
+# Vale-code for the list on https://www.plainlanguage.gov/guidelines/words/use-simple-words-phrases/
```

Comment:
Here's my Vale implementation for the [Plain language guidelines, use simple words and phrases list](https://www.plainlanguage.gov/guidelines/words/use-simple-words-phrases/).

Quote from the footer of the site:

> You may use any of the content on this site without explicit permission. As a federal website, the content is in the public domain. We ask only that you credit our work by citing back to PLAIN and www.plainlanguage.gov.

I don't know if our license is compatible with public domain?